scripts/install-nix-from-closure.sh

```diff
@@ -41,14 +41,13 @@ fi
 
 INSTALL_MODE=no-daemon
 CREATE_DARWIN_VOLUME=0
-# Trivially handle the --daemon / --no-daemon options
 while [ $# -gt 0 ]; do
     case $1 in
         --daemon)
             INSTALL_MODE=daemon;;
         --no-daemon)
             INSTALL_MODE=no-daemon;;
-        --create-volume)
+        --darwin-create-unencrypted-nix-store-volume)
```

Comment:
I intentionally made this painfully explicit to try to make 2 things roughly true:

1. Something that's likely to get copied around to/from docs, blog posts, and bootstrap scripts but not used often is _so extremely self-documenting_ that using it roughly amounts to consent. Hopefully this helps avoid stuff like:
    - People trying to use the option on other platforms showing up to complain about it not working.
    - People showing up to gripe about the volume being unencrypted.

2. There's room in the naming-system room for deciding to officially support/recommend one or more additional methods at some point (i.e., in case we do reach high confidence in a specific encryption workflow)